nixos/modules/virtualisation/podman/network-socket.nix

```diff
@@ -9,6 +9,10 @@ let
 
 in
 {
+  imports = [
+    ./network-socket-ghostunnel.nix
+  ];
```

Comment:
This import is unnecessary because it is not a dependency.
There's a dependency relation between the two modules, but it's the other way around: `network-socket-ghostunnel.nix` can not exist without `podman-network-socket.nix`, so it's actually inverted.

Reply:
Not sure I understand if you're suggesting a change or not?

Do you want it moved here with the other imports?

https://github.com/NixOS/nixpkgs/blob/96d1f311fdfad3bd7d79f4a1eb9569865d8ded17/nixos/modules/virtualisation/podman/default.nix#L41-L45